modules/tinymce/src/plugins/advlist/test/ts/browser/ContentEditableFalseTest.ts

```diff
@@ -0,0 +1,99 @@
+import { context, describe, it } from '@ephox/bedrock-client';
+import { Arr } from '@ephox/katamari';
+import { TinyAssertions, TinyHooks, TinySelections, TinyUiActions } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+import AdvListPlugin from 'tinymce/plugins/advlist/Plugin';
+import ListsPlugin from 'tinymce/plugins/lists/Plugin';
+
+interface ListStyle {
+  readonly type: string;
+  readonly style: string;
+}
+
+interface ListContents {
+  readonly listName: string;
+  readonly content: string;
+  readonly startPath: number[];
+}
+
+interface ListAction {
+  readonly title: string;
+  readonly action: (editor: Editor) => any;
+}
+
+describe('browser.tinymce.plugins.lists.ContentEditableFalseActionsTest', () => {
```

Comment:
```suggestion
describe('browser.tinymce.plugins.advlist.ContentEditableFalseActionsTest', () => {
```
This isn't a `list` test so the name should be this.